File: UctApp/ZGroupModel.cpp
//
//  ZGroupModel.cpp
//  UctApp
//
//  Created by ptyt on 14-9-4.
//  Copyright (c) 2014年 ptyt. All rights reserved.
//

#include "ZGroupModel.h"

ZGroupModel::ZGroupModel()
{
    this->users = new list<ZUserModel>();
}

ZGroupModel::~ZGroupModel()
{
    delete this->users;
}

ZGroupModel::ZGroupModel(string name,string number)
{
    this->users = new list<ZUserModel>();
    this->name = name;
    this->number = number;
}
void ZGroupModel::setName(string name)
{
    this->name = name;
}
void ZGroupModel::setNumber(string number)
{
    this->number = number;
}
string ZGroupModel::getName()
{
    return this->name;
}

string ZGroupModel::getNumber()
{
    return this->number;
}
void ZGroupModel::addUser(ZUserModel user)
{
    users->push_back(user);
}
void ZGroupModel::clearUsers()
{
    users->clear();
}

File: UctApp/ZGroupModel.swift
//
//  ZGroupModel.swift
//  UctApp
//
//  Created by ptyt on 14-9-4.
//  Copyright (c) 2014年 ptyt. All rights reserved.
//

import Foundation


class ZGroupModel:NSObject
{
    var users:NSMutableArray = NSMutableArray()
    var name:NSString?
    {
        set{
            self.name = newValue
        }
        get{
           return self.name
        }
    }
    var number:NSString?
    {
        set{
            self.number = newValue
        }get{
            return self.number
        }
    }
    
    override init()
    {
        super.init()
    }
    
    convenience init(_ name:NSString!,_ number:NSString!)
    {
        self.init()
        self.name = name
        self.number = number
    }
    
    convenience init(_ name:NSString!,_ number:NSString!,_ users:NSArray)
    {
        self.init(name,number);
        
    }
    
    func addUser(user:ZUserModel!)
    {
        users.addObject(user)
    }   
    
    
}
